README.md

```diff
@@ -55,9 +55,22 @@ HARRY HALPIN Research Scientist at Massachusetts Institute of Technology [↩](#
 ### Challenges KYODO meets
 
 - Create a framework for liquid/sustainable/ horizontal organizations, that could serve as a tool to manage its activity, be incentive and efficient. 
-- Test the new reward mode for #DAO members activity, that matches the post labor era features (fun/work, no long-term obligations, follow self-interests)
+- Test the new reward model for #DAO members activity, that matches the post labor era features (fun/work, no long-term obligations, follow self-interests)
 -  Achieve a network effect that brings mass adoption closer.
 
+## What can KYODO do?
+
+As soon as KYODO Dapp created to manage existing DAO Decent.Fund, this tool solves the real issues DF DAO met on its way. DF DAO needs are common to others DAO: many active members; self-motivation; hyper-flexibility to changing members and switching tasks; stable currency to reward contributors; effective fund management; good UX to eliminate the entry barrier for newcomers; ability to scale. We try different models and came up with the next list of features, that are already going through reality check:
```

Comment:
Not sure we can call it existing DAO, as actually we'll have DAO as soon as we implement Kyodo